### ISSUE TITLE
Redirects Not Exporting 

### ISSUE THREAD

**Issue by robertaross** [johngodley/redirection#357]
### Summary
After moving my site to the main directory on a GoDaddy Linux server from a sub directory, Redirection is not working properly). All the redirects are still working on my site (in. htaccess), but when I try to export new redirects to the .htaccess file, no redirects are shown in the text.  All the redirects are shown on your first page but they don't make it into the export file.  The file looks like nothing is in the list at all:

- Created by Redirection
- Sun, 03 Sep 2017 11:24:47 +0000
- Redirection 2.7.3 - http://urbangiraffe.com/plugins/redirection/
- IfModule mod_rewrite.c>
- /IfModule>
- End of Redirection

---------------------------------------
But here is what it looks like currently on the server (edited for brevity):

- BEGIN WordPress
- IfModule mod_rewrite.c>
- RewriteEngine On
- RewriteBase /
- RewriteRule ^index\.php$ - [L]
- RewriteCond %{REQUEST_FILENAME} !-f
- RewriteCond %{REQUEST_FILENAME} !-d
- RewriteRule . /index.php [L]
- /IfModule>
- END WordPress

- IfModule mod_rewrite.c>
- RewriteRule ^blendtecvsvitamix\.html$ /worlds-best-blenders/ [R=301,L]
- RewriteRule ^Greenstarvsangel\.html$ /greenstar-vs-super-angel-juicers/ [R=301,L]
- RewriteRule ^starting_on_raw\.html$ /starting-a-raw-food-diet/ [R=301,L]
- RewriteRule ^Products/Juicers/Juicing_FAQs/juicing_faqs\.html$ /faq/ [R=301,L]
- RewriteRule ^blendtecvsvitamix\.html$ /worlds-best-blenders/ [R=301,L]
- RewriteRule ^fasting\.html$ /how-to-enjoy-a-safe-raw-3-day-fast/ [R=301,L]
- RewriteRule ^raw_food_dehydrators\.html$ /raw-food-and-dehydrators/ [R=301,L]
- /IfModule>
- End of Redirection

**NOTE: I cut and past your content into my .htaccess file rather than use your export feature**

### Steps to reproduce
Not sure how to do this unless you want to migrate your URL from a subdir to the main directory on Linux and see what happens.

### Environment
Latest version of WordPress and Redirection hosted on WordPress.org

**Comment by johngodley** [comment]
Are you using 'view' on the Import/Export page to view these redirects, or downloading them? Does anything appear for the other export formats - JSON, CSV etc?

**Comment by robertaross** [comment]
Hi John,


Hmmm...what I was doing was exporting the Apache redirects/Apache .htaccess file, then  cut and pasting just the redirections directly into the .htacess file in my GoDaddy file manager. This was working fine until I moved my site then it stopped working. I didn't try any other settings when I first contacted you ... but now I see that I can get the redirections I need by using the WordPress Redirects/Apache .htaccess setting, then  pasting those into .htacees with GoDaddy's file manager.


Though this is fine for now, I am worried that something may still be a little bit off with your plugin since this would definitely be a workaround. And I wonder if I would be any better off using the Download option then just replacing the original file entirely???


When I download the Apache Redirects/Apache .htacess file (see attached) it has the same problem -- no redirects. Downloading WordPress Redirects/Apache .htaccess (attached) gives all the redirects.


Finally, I am attaching a copy of the .htaccess file I have been cutting and pasting to that is currently in my GoDaddy file Manager.


Hope this helps,

[1495307786965_rar_signature.png]
Robert Ross
http://www.RawFoodLife.com
[RawFoodLife.com]
________________________________
